scalecube-organization/src/main/resources/app.config.properties

```diff
@@ -0,0 +1,3 @@
+public.key.provider=io.scalecube.tokens.Auth0PublicKeyProvider
+token.expiration=2678400000
```

Comment:
token expiration should be "never" by default

meaning max long value.

Reply:
Current default value was declared previously. 
In this pull request, I've renamed config file and rounded value of token.expiration. 
Do you think it should be changed from `2678400000` (which means 31 days) to `never expires` by default?

See
https://github.com/scalecube/scalecube-organization-service/blob/master/scalecube-organization/src/main/resources/settings.properties

Btw, I just found that there is a bug in token expiration calculation.

See https://github.com/scalecube/scalecube-organization-service/pull/72

Reply:
The current default value was declared previously by Eran (@scooter72). 
In this pull request, I've renamed config file and rounded value of token.expiration. 
Do you think it should be changed from `2678400000` (which means 31 days) to `never expires` by default?

See
https://github.com/scalecube/scalecube-organization-service/blob/master/scalecube-organization/src/main/resources/settings.properties

Btw, I just found that there is a bug in token expiration calculation.

See https://github.com/scalecube/scalecube-organization-service/pull/72

Reply:
yes its will be a real problem if the token will expire after 31 days :)

Reply:
Fixed in #72 (commit https://github.com/scalecube/scalecube-organization-service/pull/72/commits/008992d5a290ae5750af3c1037fccb8c2d54e7fa)